pkg/cache/cache_test.go

```diff
@@ -1468,27 +1468,32 @@ func TestCacheQueueOperations(t *testing.T) {
 	}
 }
 
-func TestFlavorInUse(t *testing.T) {
-	rf := utiltesting.MakeResourceFlavor("x86").Obj()
-	flavor := utiltesting.MakeFlavor(rf.Name, "5").Obj()
+func TestClusterQueuesUsingFlavor(t *testing.T) {
+	x86Rf := utiltesting.MakeResourceFlavor("x86").Obj()
+	aarch64Rf := utiltesting.MakeResourceFlavor("aarch64").Obj()
+	x86Flavor := utiltesting.MakeFlavor(x86Rf.Name, "5").Obj()
+	aarch64Flavor := utiltesting.MakeFlavor(aarch64Rf.Name, "3").Obj()
```

Comment:
So what's the intension by adding `aarch64Flavor`?

Reply:
@kerthcet I added the `aarch64Flavor` since I want to verify it is okay to have unrelated ResourceFlavors.
Although, I'm ok with removing this. Should I remove  the`aarch64Flavor`?

Reply:
If you're intentioned, it's ok.